R/rlistings_methods.R

```diff
@@ -67,14 +84,23 @@ setMethod(
            nsibs = NA_integer_) {
     ## assume sortedness by keycols
     keycols <- get_keycols(tt)
+    dispcols <- listing_dispcols(tt)
```

Comment:
@gmbecker , I was wondering if we could introduce some consistency here as well. and call this "display_cols" as well?

Reply:
no, we don't want consistency here, because they actually arent' they same thing. Here, this is going to also include key columns (its the full list of all columns that will be displayed when the listing is resting), whereas the `disp_cols` arugment is does necessarily include the key columns (as that is a separate argument during construction)